test/debug/test_charptr.py

```diff
@@ -1,6 +1,6 @@
 import os
 import pytest
-from test.support import SUPPORTS_SYS_EXECUTABLE, SUPPORTS_MEM_PROTECTION
+from ..support import SUPPORTS_SYS_EXECUTABLE, SUPPORTS_MEM_PROTECTION
```

Comment:
I was already wondering because in GraalPy we have a stdlib module `test` (afaik, imported from CPython) and I needed to rename HPy tests in GraalPy. The relative import does unfortunately not work for us because we have the tests in a separate tree.

Reply:
PyPy has this layout, so the relative imports work
```
- tests
  - conftest.py  # specific for PyPY
  - support.py   # specific for PYPy
  |- pypy_tests  # specific for PyPy
  |- _vendored
   - conftest.py  # copied from hpy
   - support.py   # copied from hpy
   - test_*.py    # copied from hpy
   - | debug      # copied from hpy
   - | trace      # copied from hpy
```